### DISCUSSION TITLE
PROBLEME QNH LU SUR METAR SIMBRIEF PAS COHERENT AVEC ALTITUDE TERRAIN PFD

### DISCUSSION THREAD

**Discussion by kampinod** [flybywiresim/aircraft#4496] [Q&A]
Bonjour, depuis qqs jours je constate que lorsque je saisis dans le BARO le QNH (lu dans le METAR SIMBRIEF), l'altitude du terrain lue sur le PFD n'est pas correcte, à priori il faut saisir 1013 Hpa dans le BARO pour qu'elle soit correcte???

**Comment by CptBayer** [comment]
see other thread ... merci ...

greets Klaus

**Reply by kampinod** [reply]
merci pour  votre réponse, mais quel numéro de fil?
